docs/content/en/docs/pipeline-stages/cleanup.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Cleanup"
+linkTitle: "Cleanup"
+weight: 60
+featureId: cleanup
+---
+
+Skaffold works with [image builders]({{<relref "/docs/pipeline-stages/builders">}}) and [deployers]({{<relref "/docs/pipeline-stages/deployers">}}) that both have side effects on the environment, namely: 
+
+- resources are created in one or more namespaces in a Kubernetes cluster 
+- images are created on the local Docker daemon and registries
+    
+Skaffold offers functionality to cleanup these changes:
+
+- Kubernetes resource cleanup - `skaffold delete` command and `--cleanup=true` flag for `skaffold dev` and `skaffold debug`  
```

Comment:
umm i think `cleanup` is true by default. Maybe we should mention, if you don't want skaffold to clean up resources use `cleaup=false` in the #Kubernetes resource cleanup section.